packages/algo-ts/src/arc4/encoded-types.ts

```diff
@@ -265,14 +265,18 @@ export class Address extends Arc4ReadonlyArray<Byte> {
   get native(): Account {
     throw new NoImplementation()
   }
+
+  copy(): Address {
```

Comment:
Address shouldn't require copying as it is immutable

Reply:
It has been reverted.